packages/broker/bin/entry-point.ts

```diff
@@ -1,9 +1,9 @@
 #!/usr/bin/env node
-import { DhtNode, NodeType, PeerID } from '@streamr/dht'
+import { DhtNode, NodeType, PeerID, PeerIDKey } from '@streamr/dht'
 
 const main = async () => {
     const peerDescriptor = {
-        kademliaId: PeerID.fromString('entrypoint').value,
+        kademliaId: PeerID.fromKey('eeeeeeeeee' as PeerIDKey).value,
```

Comment:
All conversions can use `hexToBinary` and `binaryToHex` instead of `PeerID` methods

```suggestion
        kademliaId: hexToBinary('eeeeeeeeee')
```

Reply:
So should we use that conversion here?